components/Events.js

```diff
@@ -40,6 +49,15 @@ export const Events = [
 		end: new Date(2022, 9, 13, 18, 0, 0),
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Mentor Q/A",
+		location: "ACM Clubroom",
```

Comment:
WCH 127

Reply:
I'll change that. Btw, Josh did say those are potential locations not final ones but final ones will take them weeks to know if they change. This is alright for now?

Reply:
Yea thats fine!